doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,226 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-```
-
-This can be used if we want to set the data by some callable (i.e. lambda, function or [functor](https://en.wikipedia.org/wiki/Function_object#In_C_and_C++)). As with all the
-other ways, it can fail when there is no memory for the sample availabe and this failure must be handled.
-
-#### Creating a subscriber
-
-We now create a corresponding subscriber, usually in another application. Note that this will work in the same application as
-well, were it will provide the benefit of zero-copy communication, uniform usage (i.e. same syntax for inter- and 
-intraprocess communication) and lifetime management of the samples.
-
+Let's create a corresponding subscriber.
 ```cpp
 iox::popo::Subscriber<CounterTopic> subscriber({"Group", "Instance", "CounterTopic"});
-subscriber.subscribe();
 ```
-The template data type and the three string identifiers have to match those of the publisher, in other words the
-service descriptions have to be the same (otherwise we will not receive data from our publisher).
-
-We immediately subscribe here, but this can be postponed to the point where we actually want to receive data.
-
-#### Receiving data
-For simplicity we assume that we periodically check for new data. It is also possible to explicitly wait for data
-using the [Waitset](../../../iceoryx_examples/waitset/README.md). The code to receive the data is the same, the only
-difference is the way we wake up before checking for data.
-
-##### 1. Take and process a sample
-
+Now we can use the subscriber to receive data. For simplicity, we assume that we periodically check for new data. It 
+is also possible to explicitly wait for data using the 
+[Waitset](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/waitset/README.md) or the 
+[Listener](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/callbacks/README.md). The code to 
+receive the data is the same, the only difference is the way we wake up before checking for data.
 ```cpp
 while (keepRunning)
 {
     // wait for new data (either sleep and wake up periodically or by notification from the waitset)
 
-    auto result = subscriber->take();
+    auto result = subscriber.take();
 
     if(!result.has_error())
     {
-        auto& maybeSample = result.value();
-        if (maybeSample.has_value())
-        {
-            auto& sample = maybeSample.value();
-            const CounterTopic* counter = sample.get();
-            //process the data
-
-            //alternatively use operator->
-            uint32_t counter = sample->counter;
-        }
-        else
-        {
-            // we received no data
-        }
-    } else {
-        iox::popo::ChunkReceiveResult& error = result.get_error();
+        auto& sample = result.value();
+        uint32_t counter = sample->counter;
+        //process the data
+    }
+    else
+    {
         //handle the error
     }
 }
-
 ```
+By calling ``take`` we get an ``expected`` and hence we may have to handle an error.
```

Comment:
We have to handle the error or explicitly discard it with `IOX_DISCARD_RESULT` (discarding is discouraged but can be helpful for prototypes).